alameda.js

```diff
@@ -13,21 +13,31 @@ var requirejs, require, define;
   }
 
   var topReq, dataMain, src, subPath,
+    setTimeout, hasOwn, slice,
     bootstrapConfig = requirejs || require,
-    hasOwn = Object.prototype.hasOwnProperty,
     contexts = {},
     queue = [],
     currDirRegExp = /^\.\//,
     urlRegExp = /^\/|\:|\?|\.js$/,
     commentRegExp = /(\/\*([\s\S]*?)\*\/|([^:]|^)\/\/(.*)$)/mg,
     cjsRequireRegExp = /[^.]\s*require\s*\(\s*["']([^'"\s]+)["']\s*\)/g,
-    jsSuffixRegExp = /\.js$/,
-    slice = Array.prototype.slice;
+    jsSuffixRegExp = /\.js$/;
 
   if (typeof requirejs === 'function') {
     return;
   }
 
+  // Recover potentially overridden window methods from a nested browsing context
+  function cacheNativeFnReferences() {
+    var iframe = document.createElement('iframe');
+    iframe.sandbox = 'allow-same-origin';
+    document.documentElement.appendChild(iframe);
+    hasOwn     = iframe.contentWindow.Object.prototype.hasOwn;
```

Comment:
I would expect this to be `iframe.contentWindow.Object.prototype.hasOwnProperty`.